lib/rules/stylelint-disable-no-empty/index.js

```diff
@@ -0,0 +1,54 @@
+"use strict"
+
+const report = require("../../utils/report")
+const ruleMessages = require("../../utils/ruleMessages")
+const validateOptions = require("../../utils/validateOptions")
+
+const ruleName = "stylelint-disable-no-empty"
+
+const messages = ruleMessages(ruleName, {
+  rejected: "Unexpected empty `stylelint-disable` comment",
+})
+
+const stylelintDisableCommand = "stylelint-disable"
+
+const rule = (actual) => {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual })
+    if (!validOptions) {
+      return
+    }
+
+    root.walkComments((comment) => {
+      if (!isDisableCommand(comment.text)) {
+        return
+      }
+
+      const specificRules = comment.text.split(" ", 2).slice(1)
```

Comment:
> Tricky, indeed. I think this validation might actually work better not as a rule, but as a different kind of violation, like `reportNeedlessDisables`.

Agreed.

@diagramatics Sorry to mess you around. The limitations of this rule didn't become apparent until it was implemented. 

We should close this PR, and try to harvest what we can of it for @davidtheclark proposal. I'll create new separate issues for those.